lib/BouncyCheckbox.tsx

```diff
@@ -132,7 +134,8 @@ class BouncyCheckbox extends React.Component<IBouncyCheckboxProps, IState> {
     } = this.props;
     const { checked } = this.state;
     return (
-      !disableText && (
+      !disableText && 
+      (TextComponent ? <TextComponent/> : (
```

Comment:
Can we implement this instead of JSX component in here because this will be redundant. 

```jsx
  renderCheckboxText = () => {
    const {
      text,
      isChecked,
      textStyle,
      textComponent,
      textContainerStyle,
      disableBuiltInState,
      disableText = false,
    } = this.props;
    const { checked } = this.state;
    return (
      !disableText &&
      (textComponent || (
        <View style={[styles.textContainer, textContainerStyle]}>
          <Text
            style={[
              _textStyle(disableBuiltInState ? isChecked! : checked),
              textStyle,
            ]}
          >
            {text}
          </Text>
        </View>
      ))
    );
  };
```


I already have `iconComponent` implemented in the code so let's stick the same method

Reply:
@WrathChaos  kindly have a look brother on latest commit